### PR TITLE
Feat: Add short name velaapp for application CRD

### PR DESCRIPTION
**Pull request by JarHMJ**
### Description of your changes
Add short name velaapp for application CRD   (#3797)

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [x] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [x] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### Special notes for your reviewer

@wonderflow

### CODE REVIEW

**Review by barnettZQG — APPROVED** [review]
lgtm

**Review by wonderflow — APPROVED** [review]

**Comment by wonderflow** [comment]
Please run make reviewable to fix the ci

**Review by fourierr — APPROVED** [review]

**Comment by fourierr** [comment]
lgtm

File: apis/core.oam.dev/v1beta1/application_types.go [hunk 1]
```diff
@@ -82,7 +82,7 @@ type ApplicationSpec struct {
 // Application is the Schema for the applications API
 // +kubebuilder:storageversion
 // +kubebuilder:subresource:status
-// +kubebuilder:resource:categories={oam},shortName=app
+// +kubebuilder:resource:categories={oam},shortName={app,velaapp}
```
Comment (wonderflow) [inline apis/core.oam.dev/v1beta1/application_types.go:85]: why the short name didn't change the CRD file? Does it really work?

Comment (JarHMJ) [reply apis/core.oam.dev/v1beta1/application_types.go:85]:
```
go mod tidy
git --no-pager diff
diff --git a/charts/vela-core/crds/core.oam.dev_applications.yaml b/charts/vela-core/crds/core.oam.dev_applications.yaml
index 9d4fc00..82aa353 100644
--- a/charts/vela-core/crds/core.oam.dev_applications.yaml
+++ b/charts/vela-core/crds/core.oam.dev_applications.yaml
@@ -27,7 +27,6 @@ spec:
     plural: applications
     shortNames:
     - app
-    - velaapp
     singular: application
   scope: Namespaced
   versions:
diff --git a/charts/vela-minimal/crds/core.oam.dev_applications.yaml b/charts/vela-minimal/crds/core.oam.dev_applications.yaml
index 9d4fc00..82aa353 100644
--- a/charts/vela-minimal/crds/core.oam.dev_applications.yaml
+++ b/charts/vela-minimal/crds/core.oam.dev_applications.yaml
@@ -27,7 +27,6 @@ spec:
     plural: applications
     shortNames:
     - app
-    - velaapp
     singular: application
   scope: Namespaced
   versions:
git diff --quiet || (echo `date +%H:%M:%S` [FAIL] please run 'make reviewable' to include all changes && false)
19:35:09 [FAIL] please run make reviewable to include all changes
make: *** [Makefile:57: check-diff] Error 1
```
This is the result of the last CI which confuse me.  It seems that after I have changed the CRD,  the CI report the err.

**Review by wonderflow — CHANGES_REQUESTED** [review]
have you tested locally， does it really work like this?

File: apis/core.oam.dev/v1beta1/application_types.go (hunk 1 above)
Comment (wonderflow) [reply apis/core.oam.dev/v1beta1/application_types.go:85]: do you run `make reviewable`？

Comment (wonderflow) [reply apis/core.oam.dev/v1beta1/application_types.go:85]: the `make reviewable` will invoke go generate to generate CRD with the help of controller runtime. If it removes the like in the CRD, maybe that's not take affect to write like that.

**Comment by JarHMJ** [comment]
> have you tested locally， does it really work like this?

我发现了一个比较诡异的问题，我执行`go generate ./pkg/... ./apis/... `这个命令时，有时候会修改crd文件中shortNames这个字段，有时候却又不会。然后我排查是否controller-gen版本不兼容，我升级到v0.8.0时还是有这个问题。麻烦能看一下这个生成的问题么？ @wonderflow

**Comment by wonderflow** [comment]
@JarHMJ 

有两处 Application spec 要改。

You need to change both v1alpha2 and v1beta1 for application spec.

**Review by wonderflow — APPROVED** [review]

**Pull request merged**